src/containers/backdrop-library.jsx

```diff
@@ -15,15 +15,15 @@ class BackdropLibrary extends React.Component {
     }
     handleItemSelect (item) {
         const vmBackdrop = {
-            skin: `https://cdn.assets.scratch.mit.edu/internalapi/asset/${item.md5}/get/`,
             name: item.name,
             rotationCenterX: item.info[0] && item.info[0] / 2,
-            rotationCenterY: item.info[1] && item.info[1] / 2
+            rotationCenterY: item.info[1] && item.info[1] / 2,
+            skinId: null
```

Comment:
After https://github.com/LLK/scratch-vm/pull/514#discussion_r108489752 please remove this, since it won't be necessary.